test/System.IdentityModel.Tokens.Jwt.Tests/project.json

```diff
@@ -1,15 +1,19 @@
 {
     "version": "5.0.0-*",
     "dependencies": {
-        "System.IdentityModel.Tokens.Tests": "5.0.0-*"
+        "Microsoft.IdentityModel.Tokens.Tests": "5.0.0-*"
     },
     "commands": {
         "test": "xunit.runner.aspnet"
     },
     "frameworks": {
-        "dnx451": { },
+        "dnx451": {
+          "frameworkAssemblies": {
+            "System.IdentityModel": "4.0.0.0"
```

Comment:
Why adding this dependency? I don't think we are using any types in this assembly.